packages/dds/tree/docs/wip/AnchorInterfaces.ts

```diff
@@ -55,19 +59,115 @@ type RangeBoundary = {
 enum Extremity { Start, End }
 enum Side { Before, After }
 enum Tiebreak { LastToFirst, FirstToLast }
-type MovementRules = SimpleMovementRules | CustomMovementRules
-enum SimpleMovementRules { NeverMove, CommutativeMove, AlwaysMove }
-interface CustomMovementRules {
-	traitLabel: TraitLabels;
-	traitParent: TraitParents;
-	siblingStatus: NodeStatuses;
-	granularity: MoveGranularity;
-	commutative: boolean;
-}
-enum TraitLabels { Initial, Any }
-enum TraitParents { Initial, Any }
-enum NodeStatuses { Alive, Deleted, Any }
-enum MoveGranularity { IntraEdit, InterEdit, Any }
+enum MovementRules { NeverMove, CommutativeMoveInTrait, CommutativeMove }
 
 type NodeId = number;
-type TraitLabel = number;
\ No newline at end of file
+type TraitLabel = number | string;
+
+//------
+export enum ChangeType {
```

Comment:
Nit: consider using an object instead.

Reply:
These definitions were just copied over from the experimental tree DDS for the sake of discussion. I've now removed them from this file.